jcommune-service/src/test/java/org/jtalks/jcommune/service/transactional/TransactionalAuthenticatorTest.java

```diff
@@ -486,6 +488,44 @@ public void userMustHaveInitialFieldValuesWhenPluginRegistrationFailWithRegistra
         assertReflectionEquals(expectedRegisterUserDto, registerUserDto);
     }
 
+    @Test
+    public void activateAccountShouldEnableUser() throws Exception {
```

Comment:
Can we write a component test (maybe without controllers - invoking authenticator directly) that that writes to HSQLDB?